integration/hsa-integration-intyg-proxy-service/src/main/java/se/inera/intyg/infra/integration/intygproxyservice/services/authorization/HsaIntegrationAuthorizationManagementService.java

```
@@ -51,7 +53,13 @@ public List<CredentialInformation> getCredentialInformationForPerson(String pers
 
     @Override
     public HospCredentialsForPerson getHospCredentialsForPersonResponseType(String personalIdentityNumber) {
-        return null;
+        final var credentialsForPerson = getHospCredentialsForPersonService.get(personalIdentityNumber);
```

Should we create a message with the person id without hashing it?

---

After reviewing the the usage of this method I decided to remove the log & exception. This was done cause this exception is not handled, and instead, a null check is performed.

```java
 @Override
    public HospPerson getHospPerson(String personId) {

        LOG.debug("Getting hospPerson from HSA for '{}'", personId);

        HospCredentialsForPerson response = null;
        try {
            response = authorizationManagementService.getHospCredentialsForPersonResponseType(personId);
        } catch (SOAPFaultException e) {
            LOG.debug("Soap exception", e);
        }

        if (response == null || response.getPersonalIdentityNumber() == null) {
            LOG.debug("Response did not contain any hospPerson for '{}'", personId);
            return null;
        }
   }
```